components/tikv_kv/src/stats.rs

```diff
@@ -251,3 +251,23 @@ impl StatisticsSummary {
         self.count += 1;
     }
 }
+
+/// Latency indicators for multi-execution-stages.
+///
+/// The detailed meaning of the indicators is as follows:
+///
+/// ```text
+/// ------> Begin ------> Scheduled ------> SnapshotReceived ------> Finished ------>
+/// |----- schedule_wait_time -----|
+///                                |-- snapshot_wait_time --|
+/// |------------------- wait_wall_time --------------------|
+///                                                         |-- process_wall_time --|
+/// |------------------------------ kv_read_wall_time ------------------------------|
+/// ```
+#[derive(Debug, Default, Copy, Clone)]
+pub struct StageLatencyStats {
+    pub schedule_wait_time: u64,
```

Comment:
It would be better to indicate the unit in the field name.

Reply:
done